axelor-account/src/main/java/com/axelor/apps/account/web/AccountingReportController.java

```diff
@@ -54,6 +49,35 @@
 
   private final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
+  public void setFieldsFromReportTypeModelAccountingReport(
+      ActionRequest request, ActionResponse response) {
+    AccountingReport accountingReport = request.getContext().asType(AccountingReport.class);
+
+    try {
+      AccountingReportType accountingReportType =
+          Beans.get(AccountingReportTypeRepository.class)
+              .find(accountingReport.getReportType().getId());
+      logger.debug("accountingReportType : {}", accountingReportType);
+      if (accountingReportType.getModelAccountingReport() != null) {
+        AccountingReport accountingReport1 =
+            JPA.copy(accountingReportType.getModelAccountingReport(), false);
+        accountingReport1.setRef(null);
+        accountingReport1.setStatusSelect(null);
+        accountingReport1.setPublicationDateTime(null);
+        accountingReport1.setTotalCredit(null);
+        accountingReport1.setTotalDebit(null);
+        accountingReport1.setBalance(null);
+        accountingReport1.setDate(accountingReport.getDate());
+        accountingReport1.setDateTo(accountingReport.getDateTo());
```

Comment:
this line should be deleted. We want keep the dateTo value of the model, like dateFrom field